cmd/eelbot/main.go

```diff
@@ -16,10 +17,21 @@ import (
 	"github.com/emseers/eelbot"
 	"github.com/emseers/eelbot/commands"
 	"github.com/emseers/eelbot/replies"
-	_ "github.com/mattn/go-sqlite3"
+	_ "github.com/jackc/pgx/v5/stdlib"
 	"gopkg.in/yaml.v3"
 )
 
+func mustDo(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func must[val any](v val, err error) val {
+	mustDo(err)
+	return v
+}
+
```

Comment:
I feel like this change could have been a separate PR so as to keep this one more straightforward to read through.

Reply:
TBH, I don't know if this is even idiomatic go. It just seemed like a change to allow for tighter code.